src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -1019,4 +1020,18 @@ public void onInventoryPickupItem(InventoryPickupItemEvent event)
             }
         }
     }
+    
+    @EventHandler(ignoreCancelled = true)
+    public void onItemFrameBrokenByBoat(final HangingBreakEvent event)
+    {
+        // Checks if the event is caused by Physic damage - 90% of cases caused by a boat (other 10% would be block, 
+        // however since it's in a claim, unless you use a TNT block we can not worry about it).
+        if (event.getCause() != HangingBreakEvent.RemoveCause.PHYSICS) 
+        {
+            return;
+        }
+        
+        // Cancels the event in any case, as we can not efficiently retrieve the person/entity who broke the Item Frame/Hangable Item.
```

Comment:
But we can determine where it's happening? e.g. allow if not within a claim?

Reply:
You could, but I feel like it's unnecessary since it's an odd mechanic, to begin with.